src/services/API/sqs.js

```diff
@@ -0,0 +1,50 @@
+import { SQSClient, SendMessageCommand } from "@aws-sdk/client-sqs";
+
+const REGION = "ap-south-1";
+const queueURL = "https://sqs.ap-south-1.amazonaws.com/111766607077/EventQueue";
+const sqsClient = new SQSClient({
+  region: REGION,
+  credentials: {
+    accessKeyId: 'AKIARUBOPCTSSU3W7SML',
+    secretAccessKey: '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',
+  },
+});
```

Comment:
Make this into environment variables as soon as possible as we might be getting a mail from AWS
- create one `.env.example` if there isn't already and update the documentation file ENV.md (create one if it doesn't exist)
- You can see plio's repo for an example
- For continuous deployment, you'll have to add these information as github secrets (you try once, let me know if it isn't working)